src/layout/section/cardsStackSection/CardsStackSection.story.mdx

```diff
@@ -0,0 +1,40 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { TripCard } from '../../../tripCard'
+import { tripCardConfig } from './CardsStackSection.example'
+import { CardsStackSection } from './index'
+
+<Meta title="Sections/CardsStackSection" />
+
+
+# **CardsStackSection**
+
+<Canvas>
+  <Story name="default">
+    <CardsStackSection>
+      <TripCard {...tripCardConfig} />
+      <TripCard {...tripCardConfig} />
+      <TripCard {...tripCardConfig} />
+      <TripCard {...tripCardConfig} />
+      <TripCard {...tripCardConfig} />
+      <TripCard {...tripCardConfig} />
+    </CardsStackSection>
+  </Story>
+</Canvas>
+
+## Specifications
+
+Used to display stacked TripCards.
```

Comment:
IMHO it can be applied to stack any list of components. 

Don't really understand why you need it instead of applying **spacing rules**. We should avoid having new components that are nor Pixar nor Product for now... 

Or... maybe you are treating this as a "Layout"?? hehehe

Reply:
I explained it this morning during stand-up: to normalize TripCards, we can either:

- Put margins on the `TripCard` elements. We can't use the default `normalizeHorizontally` method as it's adding padding, and here it's meant to be outside the component. Moreover, adding those will completely mess up the spacings in `CardsSection` and `CardsGridSection` that we're using (and we'll have to override everything)
- In fact, search results and YourRides are the only cards not using a section, so I though "why not creating a section for those also". On the plus side, no override is needed on existing sections. 

I got approvals from most of the team then.